src/main/java/pl/connectis/Spotify_API_CLI/authorization/Token.java

```diff
@@ -0,0 +1,100 @@
+package pl.connectis.Spotify_API_CLI.authorization;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.*;
+
+@Getter
+@Setter
+@ToString
+@EqualsAndHashCode
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Slf4j
+public class Token implements Serializable {
```

Comment:
Mix DTO i beanu springowego - trzeba to rozdzielić bo nie może tak być że tworzysz beana i rownoczesnie RestTemplate Ci zwraca inna instancje której parametry wpisujesz w beana... totalnie to zagmatwane. Metody save/read powinny być cześcią jakiegos `TokenService` `Token` powinien być beanem i dodatkowo powinieneś miec jeszcze `TokenDTO`

Reply:
Ok, sprobuje zmienic tak jak proponujesz, moze bedzie latwiej.